src/components/Interpretations/InterpretationsUnit/InterpretationsUnit.js

```diff
@@ -31,96 +37,151 @@ const interpretationsQuery = {
+                        }
+
+                        100% {
+                            transform: rotate(360deg);
+                        }
+                    }
+
+                    .expanded {
+                        padding-bottom: ${spacers.dp32};
+                    }
+
+                    .loader {
+                        display: flex;
+                        justify-content: center;
+                    }
+
+                    .header {
+                        display: flex;
+                        justify-content: space-between;
+                    }
+
+                    .title {
+                        font-size: ${spacers.dp16};
+                        font-weight: 500;
+                        line-height: 21px;
+                        color: ${colors.grey900};
+                    }
+                `}</style>
             </div>
-            {isExpanded && (
-                <>
-                    {loading && (
-                        <div className="loader">
-                            <CircularLoader small />
-                        </div>
-                    )}
-                    {data && (
-                        <>
-                            <InterpretationList
-                                currentUser={currentUser}
-                                interpretations={
-                                    data.interpretations.interpretations
-                                }
-                                onInterpretationClick={onInterpretationClick}
-                                refresh={onCompleteAction}
-                            />
-                            <InterpretationForm
-                                currentUser={currentUser}
-                                type={type}
-                                id={id}
-                                onSave={onCompleteAction}
-                            />
-                        </>
-                    )}
-                </>
-            )}
-            <style jsx>{`
-                .container {
-                    padding: ${spacers.dp16};
-                    border-bottom: 1px solid ${colors.grey400};
-                    background-color: ${colors.white};
-                }
-
-                .expanded {
-                    padding-bottom: ${spacers.dp32};
-                }
-
-                .loader {
-                    display: flex;
-                    justify-content: center;
-                }
-
-                .header {
-                    display: flex;
-                    justify-content: space-between;
-                }
-
-                .title {
-                    font-size: ${spacers.dp16};
-                    font-weight: 500;
-                    line-height: 21px;
-                    color: ${colors.grey900};
-                }
-            `}</style>
-        </div>
-    )
-}
+        )
+    }
+)
+
+InterpretationsUnit.displayName = 'InterpretationsUnit'
```

Comment:
Is this to avoid the linting error when importing default?

Reply:
All components that are wrapped in `forwardRef` should have a `displayName`, otherwise they show up in React DevTools as `Component`. In most DHIS2 projects not adding the `displayName` will result in an ESLint error, I'm not sure if I got one here, or if I simply added it out of habbit. In any case, it will help when debugging stuff in DevTools and will also help in Jest tests.